appveyor.yml

```diff
@@ -14,21 +16,25 @@ matrix:
 # what combinations to test
 environment:
   matrix:
-    - nodejs_version: 8
+    - nodejs_version: 10
```

Comment:
The CLI is also using balena CI's own VMs to run tests other versions of Node. While we rely on Appveyor to build the native installers using Node 10, it makes sense to make sure that the tests pass on Appveyor for Node 10.